ProxyAgent/HttpClient/HttpRequest.cs

```diff
@@ -147,5 +147,12 @@ public IHttpRequest SetContent<T>(T sourceObject, Encoding encoding, MediaTypeHe
             this.ContentType = mediaType;
             return this;
         }
+
+        public IHttpRequest SetContent(byte[] content, string mediaType)
```

Comment:
Why not use one of the generics above SetContent<T>?

Reply:
Those sets the content as a StringContent, and the content should be ByteArrayContent